src/Compilers/CSharp/Portable/Symbols/Tuples/TupleFieldSymbol.cs

```diff
@@ -11,38 +11,61 @@
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     /// <summary>
-    /// Represents a non-element field of a tuple type (such as (int, byte).Rest)
-    /// that is backed by a real field within the tuple underlying type.
+    /// A plain TupleElementFieldSymbol (as opposed to a TupleVirtualElementFieldSymbol) represents
+    /// an element field of a tuple type (such as (int, byte).Item1) that is backed by a real field
+    /// with the same name within the tuple underlying type.
+    ///
+    /// Note that original tuple fields (like 'System.ValueTuple`2.Item1') do not get wrapped.
     /// </summary>
-    internal class TupleFieldSymbol : WrappedFieldSymbol
+    internal class TupleElementFieldSymbol : WrappedFieldSymbol
```

Comment:
>TupleElementFieldSymbol [](start = 19, length = 23)

:memo: I merged TupleFieldSymbol into TupleElementFieldSymbol, as we no longer have a need for TupleFieldSymbol per-se (we're not wrapping custom fields anymore).